### ISSUE TITLE
TLS validation

### ISSUE THREAD

**Issue by ioggstream** [aaronpk/oauth-v2-1#23]
## Question

7.4.2 states

```the client MUST validate the TLS
   certificate chain when making requests to protected resources,
   including checking the Certificate Revocation List (CRL) [RFC5280].
```

While this is correct, there are various way of addressing TLS security, eg. https://httpwg.org/specs/rfc8740.html

Probably we should find a way to start from a generic TLS BCP and extend on that.

**Comment by ioggstream** [comment]
@aaronpk WDYT ?

**Comment by ioggstream** [comment]
the validation issue has been fixed directly in https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html

closed in favor of #50
